test/runtests.jl

```diff
@@ -43,8 +43,9 @@ end
 for meth in [:jprod!, :jtprod!]
   @eval @test_throws(MethodError, $meth(model, [0.0], [1.0], [2.0]))
 end
-@test_throws(MethodError, jth_hprod(model, [0.0], [1.0], 2))
+@test_throws(AssertionError, jth_hprod(model, [0.0], [1.0], 2))
```

Comment:
I don't think this should be here, since it's supposed to test `MethodError`s. I'm guessing the argument being outside of the scope is the issue. Perhaps changing `DummyModel` a bit will be enough.